controller_manager:
  ros__parameters:
    joint_trajectory_controller:
      type: "joint_trajectory_controller/JointTrajectoryController"
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

forward_position_controller:
  type: forward_command_controller/ForwardCommandController

forward_position_controller:
  ros__parameters:
    joints:
      - BR1

    interface_name: position
